contracts/optimism/TokenRateOracle.sol

```diff
@@ -147,25 +163,24 @@ contract TokenRateOracle is CrossDomainEnabled, ITokenRateOracle, Versioned {
         uint256 newTokenRate_, uint256 newRateL1Timestamp_
     ) internal view returns (bool) {
```

Comment:
:warning:  should we plug here `MIN_ALLOWED_TOKEN_RATE` instead of `0` for the bottom check
and, likewise, `MAX_ALLOWED_TOKEN_RATE` to limit the `topTokenRateLimit` value?